src/chrome/content/rules/Zillow.com.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Non-functional hosts
+		Timeout was reached:
+		- engineering.zillow.com
+
+		Certificate mismatched:
+		- investors.zillow.com
+
+		Status code mismatch:
+		- mortgageapi.zillow.com
+
+		Mixed content blocking (MCB) triggered:
+     		- info.zillow.com
```

Comment:
Could you also provide an example of an URL where you get MCB please? I simply get a (302->)404 on https://info.zillow.com/